package.json

```
@@ -70,6 +70,7 @@
       "trim@<0.0.3": ">=0.0.3",
       "node-forge@<1.3.0": ">=1.3.0",
       "minimist@<=1.2.5": ">=1.2.6",
+      "@pantheon-systems/wordpress-kit": "workspace:^1.0.0",
       "@pantheon-systems/drupal-kit": "workspace:^1.0.0",
```

Let's also make this change here, since trying to run the starters in the monorepo context will fail as is.
```suggestion
      "@pantheon-systems/wordpress-kit": "workspace:*",
      "@pantheon-systems/drupal-kit": "workspace:*",
```